recipes/bluesky/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "bluesky" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "e68dbcb64f98997a193408d1510c70d0ee00dedf79c35dad703f35d773f02e11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/NSLS-II/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - cycler
+    - event-model
+    - historydict
+    - ipython
+    - jsonschema
+    - lmfit
+    - matplotlib
+    - numpy
+    - super_state_machine
+    - toolz
+    - tqdm
+    - pyzmq
+
+test:
+  imports:
+    - bluesky
+    - bluesky.plans
+    - bluesky.callbacks
+
+about:
+  home: https://github.com/NSLS-II/bluesky
+  license: BSD
```

Comment:
This looks like a `BSD-3-Clause`

Reply:
I think it is a BNL modified 3-Clause do you want as 3-Clause or something else to capture the mod?